robot/power.py

```diff
@@ -43,6 +56,15 @@ def power_off(self):
         """
         self._send_and_receive({'power': False})
 
+    def set_output(self, output: PowerOutput, value: bool) -> None:
+        """
+        Turn off individual power output.
+        """
+        self._send_and_receive({
+            'power-output': output.value,
+            'power-level': value,
+        })
```

Comment:
Can we typecheck these? Passing in 0.5 as the power level to get 50% power is the type of thing which teams would attempt and that'll produce a very odd error message at the mo.